ci/release/pypi_uploader.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+set -e
+
+
+SOURCE_BRANCH=master
+
+# Skip upload if CI is executed locally
+if [[ ${RUNNING_CI_LOCALLY} = true  ]]; then
+    echo "Skipping PyPi upload - running ci locally"
+    return 0
+# Skip upload if CI is not executed locally and current branch is not master
+elif [ "${GIT_BRANCH}" != "${SOURCE_BRANCH}" ]; then
+    echo "Skipping PyPI upload - not master branch"
+    #return 0
```

Comment:
Uncomment the `return 0` since we only want this running for `master` branch

You're already of aware of this one, but just flagging it for completeness :)